src/tests/multiplexer-integration-test.ts

```diff
@@ -86,11 +116,15 @@ describe('Multiplexer', () => {
         .expectRenderSlot('List', 'root', {contentTypes: ['templateName', 'model']})
         .expectRenderSlot('PostMuxer', 'item', {contentTypes: ['templateName', 'model']})
         .expectRenderSlot('ShowOne', 'item', {contentTypes: ['templateName', 'model']})
-        .expectRenderSlot('PostMuxer', 'item', {contentTypes: ['templateName', 'model']});
+        .expectRenderSlot('PostMuxer', 'item', {contentTypes: ['templateName', 'model']})
+        // Current implementation of multiplexer is to update all the existing hosted particle inputs.
+        .expectRenderSlot('ShowOne', 'item', {contentTypes: ['template', 'templateName', 'model'], times: 2})
+        .expectRenderSlot('ShowTwo', 'item', {contentTypes: ['template', 'templateName', 'model']});
     const postsStore = await collectionHandleForTest(arc, arc.findStoreById(arc.activeRecipe.handles[0].id));
     await postsStore.add(
         Entity.identify(new postsStore.entityClass({message: 'w', renderRecipe: recipeOne, renderParticleSpec: showOneSpec}), '4'));
     await arc.idle;
+    await arc.idle;
```

Comment:
I think i saw a TODO somewhere to investigate the double `idle` calls, can you, please, file a bug, so that we track it better? maybe even temporarily do the double waiting inside arc's idle method?
I'm worried that we only do the double wait in tests, but not in production code and it'll not work as expected when we migrate to storage NG.

Reply:
resolved offline